### ISSUE TITLE
Fork of Transformers

### ISSUE THREAD

**Issue by dashstander** [XiangLi1999/Diffusion-LM#1]
Hey! Thanks so much for making your code available. I'm following the set up right now, but could you perhaps talk about why you're using a fork of the Huggingface Transformers repo? I imagine that you made some changes to it, if so could you point out what and where they are? 

Thanks again!

**Comment by XiangLi1999** [comment]
Hi, 
Thanks for the question. 
I think for this version, I didn't change the library part, so it shouldn't affect the Diffusion-LM code if you just use your original transformer install. (There is another version where I changed the generation_util.py to implement PoE decoding, but it's not here.)

I changed some code here: ``Diffusion-LM/transformers/examples/pytorch/language-modeling/*`` to  implement some baselines and classifier models.

**Comment by dashstander** [comment]
Thanks so much @XiangLi1999 ! So is it safe to say that, for those of us who just want to reproduce your model we can just install the `transformers` from pip or conda and be fine?

**Comment by cameronalonso2** [comment]
Hi @XiangLi1999 
this is a bad idea to fork the codes of another library, and make your codes super untidy, this is not clear which files have been changed and hard to follow the method implementation. please do NOT fork a library, and modify the files and install the original library.

**Comment by rabeehk** [comment]
Hi @XiangLi1999 
thanks for sharing the codes. About "There is another version where I changed the generation_util.py to implement PoE decoding, but it's not here." so you mean there is some parts of the codes which does not exist in the repo? I greatly appreciate if you could share all parts of the codes to be able to reproduce the results.
Many thanks

**Comment by rschaffar** [comment]
For everyone who wants to see the detailed differences. I searched for the commit with the minimum amount of differences to the forked code in the project.

Apparently this was forked from commit 8ce133063120683018b214fe10d1449e4c2401da of https://github.com/huggingface/transformers
